packages/core/src/http-handler.ts

```diff
@@ -60,6 +62,22 @@ export class HttpHandler {
     return requestContext;
   }
 
+  protected _bindGetFromContext(context: Context): void {
+    context.bind('getFromContext').toDynamicValue(() => {
+      return async (key: string) => {
+        return await context.get(key);
+      };
+    });
+  }
+
+  protected _bindBindElement(context: Context): void {
+    context.bind('bindElement').toDynamicValue(() => {
+      return async (key: string, value: ValueOrPromise<BoundValue>) => {
```

Comment:
This does not support all different Binding styles (static value, class constructor). Also `element` is a new term that was not used anywhere else so far, I'd rather leave it out from the binding name.

```ts
context.bind('bind').to((key: string): Binding => context.bind(key));
```